config/environments/development.rb

```diff
@@ -1,6 +1,15 @@
 require "active_support/core_ext/integer/time"
 
 Rails.application.configure do
+  config.after_initialize do
+    Bullet.enable        = true
+    Bullet.alert         = true
```

Comment:
Keep this false.

Reply:
hi @rt4914 i kept both false now. thanks